app/templates/components/upcoming-project-card.hbs

```diff
@@ -0,0 +1,130 @@
+<div
+  class="project-summary-card callout"
+  data-test-project-card
+>
+  <div class="grid-x grid-x-small-gutters">
+    <div class="cell large-5">
+      <h3 class="tiny-margin-bottom">
+        {{#link-to 'show-project' project.id}}{{project.dcpProjectname}}{{/link-to}}
+        <small class="dark-gray">{{project.dcpUlurpNonulurp}}</small>
+      </h3>
+      <h5 class="applicant">{{project.applicants}}</h5>
+      <p>{{project.dcpProjectbrief}}</p>
+      {{#if (and hearingsWaived (not hearingsSubmitted))}}
+        <span data-test-hearings-waived-message="{{project.id}}">You have opted out of submitting hearings</span>
+      {{/if}}
+      {{#if (and hearingsSubmitted (not hearingsWaived))}}
```

Comment:
Yeah, I'd also have these sorts of complicated logical states also as computeds in the component file, just because it can get confusing to reason abotu.